dspy/evaluate/metrics.py

```diff
@@ -9,28 +9,97 @@
 
 
 def EM(prediction, answers_list):  # noqa: N802
-    assert isinstance(answers_list, list)
+    """Return True if any reference exactly matches the prediction (after normalization).
+
+    Normalization includes: Unicode NFD, lowercasing, punctuation removal,
+    English article removal ("a", "an", "the"), and whitespace collapse.
+
+    Args:
+        prediction (str): Predicted answer string.
+        answers_list (list[str]): List of reference answers.
 
+    Returns:
+        bool: True if any reference exactly equals the prediction after normalization;
+            otherwise False.
+
+    Example:
+        >>> EM("The Eiffel Tower", ["Eiffel Tower", "Louvre"])
+        True
+        >>> EM("paris", ["Paris"])
+        True
+    """
+    assert isinstance(answers_list, list)
```

Comment:
let's don't mix fix with docstring changes. Actually this `assert` statement won't provide more information to users